Doc/reference/expressions.rst

```diff
@@ -1308,8 +1319,10 @@ The shifting operations have lower priority than the arithmetic operations:
 .. productionlist:: python-grammar
    shift_expr: `a_expr` | `shift_expr` ("<<" | ">>") `a_expr`
 
-These operators accept integers as arguments.  They shift the first argument to
-the left or right by the number of bits given by the second argument.
+These operators accept integers as arguments or custom objects that override
+:meth:`__lshift__` and :meth:`__rshift__` special methods.  In the former case they
+shift the first argument to the left or right by the number of bits given by the
+second argument.
```

Comment:
Ditto, don't mix advanced information with the basic information. It gets confusing.